src/Backend/ServiceFactory.php

```diff
@@ -63,6 +66,11 @@ protected function createService($serviceName, array $config)
     {
         if (isset($this->serviceMap[$serviceName])) {
             $service = $this->serviceMap[$serviceName];
+        } elseif (
+            in_array($serviceName, $this->deletedServices)
+            || !file_exists(__DIR__ . '/' . $serviceName . '.php')
+        ) {
+            throw new \InvalidArgumentException('Invalid service name "' . $serviceName . '".');
```

Comment:
Note that this exception is caught by the calling function `getServicesByName`. The error message is added here in case if there will be some logging added with future pull requests.